### PR TITLE
Initial Import

### PR DESCRIPTION
**Pull request by andrewvc**

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Contributors:
+* Colin Surprenant (colinsurprenant)
+* Jordan Sissel (jordansissel)
+* João Duarte (jsvd)
+* Kurt Hurtado (kurtado)
+* Nick Ethier (nickethier)
+* Pier-Hugues Pellerin (ph)
+* Richard Pijnenburg (electrical)
+* Tal Levy (talevy)
```
Comment (untergeek) [inline CONTRIBUTORS]: What?!  How am I not in this list? #mocksurprise...

File: lib/logstash/codecs/nmap.rb [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+  # Emit each hop_tuple of the traceroute with type 'nmap_traceroute_link'
+  config :emit_traceroute_links, :validate => :boolean, :default => true
+
+  public
+  def register
+  end
+
+  public
+  def decode(data)
+    xml = Nmap::XML.parse(data)
```
Comment (untergeek) [inline lib/logstash/codecs/nmap.rb:25]: Well, that sure makes it easier to parse the XML :smile:

**Comment by untergeek** [comment]
This is exceptionally clean code.  All tests pass. LGTM™

**Comment by andrewvc** [comment]
@untergeek SWEEEEET!

**Pull request closed without merging**